packages/static-server/lib/server.js

```diff
@@ -0,0 +1,118 @@
ion(n,i){n?e._readdirError(t,n,r):e._readdirEntries(t,i,r)}}(this,e,r))}},E.prototype._readdirEntries=function(e,t,r){if(!this.aborted){if(!this.mark&&!this.stat)for(var n=0;n<t.length;n++){var i=t[n];i="/"===e?e+i:e+"/"+i,this.cache[i]=!0}return this.cache[e]=t,r(null,t)}},E.prototype._readdirError=function(e,t,r){if(!this.aborted){switch(t.code){case"ENOTSUP":case"ENOTDIR":var n=this._makeAbs(e);if(this.cache[n]="FILE",n===this.cwdAbs){var i=new Error(t.code+" invalid cwd "+this.cwd);i.path=this.cwd,i.code=t.code,this.emit("error",i),this.abort()}break;case"ENOENT":case"ELOOP":case"ENAMETOOLONG":case"UNKNOWN":this.cache[this._makeAbs(e)]=!1;break;default:this.cache[this._makeAbs(e)]=!1,this.strict&&(this.emit("error",t),this.abort()),this.silent||console.error("glob error",t)}return r()}},E.prototype._processGlobStar=function(e,t,r,n,i,o,a){var s=this;this._readdir(r,o,(function(l,c){s._processGlobStar2(e,t,r,n,i,o,c,a)}))},E.prototype._processGlobStar2=function(e,t,r,n,i,o,a,s){if(!a)return s();var l=n.slice(1),c=e?[e]:[],u=c.concat(l);this._process(u,i,!1,s);var p=this.symlinks[r],h=a.length;if(p&&o)return s();for(var f=0;f<h;f++){if("."!==a[f].charAt(0)||this.dot){var d=c.concat(a[f],l);this._process(d,i,!0,s);var m=c.concat(a[f],n);this._process(m,i,!0,s)}}s()},E.prototype._processSimple=function(e,t,r){var n=this;this._stat(e,(function(i,o){n._processSimple2(e,t,i,o,r)}))},E.prototype._processSimple2=function(e,t,r,n,i){if(this.matches[t]||(this.matches[t]=Object.create(null)),!n)return i();if(e&&u(e)&&!this.nomount){var o=/[\/\\]$/.test(e);"/"===e.charAt(0)?e=l.join(this.root,e):(e=l.resolve(this.root,e),o&&(e+="/"))}"win32"===process.platform&&(e=e.replace(/\\/g,"/")),this._emitMatch(t,e),i()},E.prototype._stat=function(e,t){var r=this._makeAbs(e),i="/"===e.slice(-1);if(e.length>this.maxLength)return t();if(!this.stat&&d(this.cache,r)){var o=this.cache[r];if(Array.isArray(o)&&(o="DIR"),!i||"DIR"===o)return t(null,o);if(i&&"FILE"===o)return t()}var a=this.statCache[r];if(void 0!==a){if(!1===a)return t(null,a);var s=a.isDirectory()?"DIR":"FILE";return i&&"FILE"===s?t():t(null,s,a)}var l=this,c=m("stat\0"+r,(function(i,o){if(o&&o.isSymbolicLink())return n.stat(r,(function(n,i){n?l._stat2(e,r,null,o,t):l._stat2(e,r,n,i,t)}));l._stat2(e,r,i,o,t)}));c&&n.lstat(r,c)},E.prototype._stat2=function(e,t,r,n,i){if(r&&("ENOENT"===r.code||"ENOTDIR"===r.code))return this.statCache[t]=!1,i();var o="/"===e.slice(-1);if(this.statCache[t]=n,"/"===t.slice(-1)&&n&&!n.isDirectory())return i(null,!1,n);var a=!0;return n&&(a=n.isDirectory()?"DIR":"FILE"),this.cache[t]=this.cache[t]||a,o&&"FILE"===a?i():i(null,a,n)}},function(e,t,r){e.exports=u,u.realpath=u,u.sync=p,u.realpathSync=p,u.monkeypatch=function(){n.realpath=u,n.realpathSync=p},u.unmonkeypatch=function(){n.realpath=i,n.realpathSync=o};var n=r(5),i=n.realpath,o=n.realpathSync,a=process.version,s=/^v[0-5]\./.test(a),l=r(298);function c(e){return e&&"realpath"===e.syscall&&("ELOOP"===e.code||"ENOMEM"===e.c
```

Comment:
Is it added by mistake?

Reply:
yes definitely :-) I just notice that as well.
i'm adding this file into .gitignore